lua/various-textobjs/charwise-textobjs.lua

```diff
@@ -351,6 +351,32 @@ function M.mdlink(scope, lookForwL)
 	selectTextobj(pattern, scope, lookForwL)
 end
 
+---@param scope "inner"|"outer" inner selector only includes the content, outer selector includes the type.
+---@param lookForwL integer
+function M.mdEmphasis(scope, lookForwL)
+	local patterns = {}
+	for _, tag in ipairs({ "*", "**", "***", "_", "__", "___" }) do
+		local tagChar = tag:sub(1, 1)
+		local escapedTag = tag
+		if tagChar == "*" then
```

Comment:
You could use `vim.pesc` to simplify this part: https://neovim.io/doc/user/lua.html#vim.pesc()